raster3d/r3.in.v5d/v5d.c

```diff
@@ -63,6 +63,7 @@
 #include <string.h>
 #include <math.h>
 #include <grass/gis.h>
+#include <errno.h>
 #include "binio.h"
```

Comment:
Please, keep the empty line between include groups:

```suggestion
#include <errno.h>

#include "binio.h"
```